src/components/Trade/Components/TradeQuotes/useCountDown.ts

```diff
@@ -0,0 +1,23 @@
+import { bnOrZero } from '@shapeshiftoss/investor-foxy'
+import { useEffect, useState } from 'react'
+export function useCountdown(mins: number) {
```

Comment:
Ah, this is the countdown stuff you mentioned - we might want to do this in a seperate PR, as it'll take a bit of thinking to get right.